app/models/user.rb

```diff
@@ -104,9 +104,9 @@ def owned_by(user)
 
   DEFAULT_FEATURE_FLAGS = {
     # whether the home page uses React and the new search API
-    "advanced_search" => false,
+    "advanced_search" => true,
     # whether the facet search controls are shown
-    "faceted_search" => false,
+    "faceted_search" => true,
```

Comment:
blocker: i think this was left in accidentally

Reply:
Good catch!  Fixed.